src/ssl.c

```diff
@@ -7922,6 +7923,67 @@ WOLFSSL_X509_EXTENSION* wolfSSL_X509_set_ext(WOLFSSL_X509* x509, int loc)
                 ext->crit = x509->CRLdistCrit;
                 break;
 
+            case ALT_NAMES_OID:
+                if (!isSet)
+                    break;
+
+                WOLFSSL_GENERAL_NAME* gn = NULL;
```

Comment:
Please move variable declarations to top of case. You'll need to add braces between case : and break;